Reference/Security/Security-hardening/index.md

```diff
@@ -48,6 +48,8 @@ Some older versions of Umbraco also relied on /umbraco/webservices/ for loadbala
 
 If your server is behind a load balancer, you should use `{HTTP_X_FORWARDED_FOR}` instead of `{REMOTE_ADDR}` as the input for the rule.
 
+In some server configurations you will need to allow the server that is running the Umbraco backoffice to be able to make internal requests to the /umbraco folder. To do that, you will need to add the servers IP address to the rewrite rule as negations.
```

Comment:
Looks like something that was added by mistake? 🤔

Reply:
uhh I do not know how that got there- I did not write that

Reply:
can I just delete that file and all would be good?

Reply:
Strange. 
You should remove that part of the file, not the entire file.
Removing the entire file, will also remove it from the docs 🙈